packages/pds/src/services/feed/index.ts

```diff
@@ -108,6 +121,8 @@ export class FeedService {
       .selectFrom('post')
       .where('post.uri', 'in', postUris)
       .innerJoin('ipld_block', 'ipld_block.cid', 'post.cid')
+      .innerJoin('did_handle', 'did_handle.did', 'post.creator')
+      .where(actorNotSoftDeletedClause())
```

Comment:
Reminder to self: take a look at this— possible we want to keep this logic out of `getPostViews()` and relegate it entirely to `selectPostQb()`, etc.

Reply:
Keeping this, as it covers taken-down replies on the feed views.  Added a note in the code to make that clear.